azext_iot/_params.py

```diff
@@ -958,6 +979,11 @@ def load_arguments(self, _):
             help="The secondary symmetric shared access key stored in base64 format. ",
             arg_group="Authentication"
         )
+        context.argument(
+            "device_information",
+            options_list=["--device-information", "--info"],
+            help="Optional device information.",
```

Comment:
Ideally we could provide a bit more detail, but this is something service docs should cover.

Reply:
It has the same format as tags, which is shown in the help example.

Reply:
I'm talking about the business use case. Just knowing the format doesn't cover this.